lang/rs/bindings/cne-sys/build.rs

```diff
@@ -16,7 +16,7 @@ fn main() {
     let (cndp_lib_dir, cndp_include_dir) = match cndp_install_path {
         Some(install_path) => (
             install_path.to_string() + &default_lib_dir,
-            install_path.to_string() + &default_include_dir,
+            install_path + &default_include_dir,
```

Comment:
This is odd. I don't konw how the rust stuff got in this PR. I see you merged, but I wonder if you cherry-picked instead? I think GitHub will be able to figure it out. Let's see.

Reply:
I was most likely me, I was trying to update the commit and not do a force merge and I remember a cherry pick.